canvas_modules/common-canvas/src/common-canvas/svg-canvas-d3.js

```diff
@@ -147,8 +160,11 @@ export default class SVGCanvasD3 {
 
 					} else if (CanvasUtils.isCmndCtrlPressed(d3Event) && d3Event.keyCode === V_KEY && actions.pasteFromClipboard) {
 						CanvasUtils.stopPropagationAndPreventDefault(d3Event);
-						this.canvasController.keyboardActionHandler("paste");
-
+						if (this.mousePos) { this.mousePos = this.renderer.convertPageCoordsToCanvasCoords(this.mousePos.x, this.mousePos.y);
+							this.mousePos = this.renderer.getMousePosSnapToGrid(this.mousePos); this.canvasController.keyboardActionHandler("paste", this.mousePos);
```

Comment:
Why are these lines of code on the same line? Semi-colon should indicate the end of a line of code - next line of code is on the following line in the editor.